content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -283,9 +298,18 @@ set -ex
 psql file.sql
 ```
 
+ 
+</tab> 
+
+ 
+</TabbedContent>
+
+
 ## Integrated seeding with Prisma Migrate
 
-Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. Prisma Migrate resets the database and triggers seeding in the following scenarios:
+Prisma Migrate integrates seamlessly with your seeds, assuming you follow the steps in the previous section. When Prisma Migrate resets the development database, seeding is triggered automatically. When Prisma Migrate resets the development database, seeding is triggered automatically. 
```

Comment:
Nitpick: the last sentence is repeated here

Reply:
Thats not a nitpick! You're totally right to point that out! 🙌